source/expressions.tex

```diff
@@ -3637,6 +3637,12 @@
 of the program. Whether or not the destructor is called for the
 \tcode{std::type_info} object at the end of the program is unspecified.
 
+\pnum
+If the \grammarterm{expression} operand is of class type, or
```

Comment:
(possible cv-qualified) in both cases is significant here.

Reply:
How so? A cv-qualified class type is also a class type.

Reply:
Since in [basic.types#general] p9, "cv-qualified versions of these types" are all used in these cases, which seems class type does not cover "cv-qualified versions of class type", they distinct.

Reply:
Yeah, [basic.types] is not always good about these things.

Added "possibly cv-qualified".